jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/NotificationUtils.java

```diff
@@ -226,7 +226,7 @@ public static void triggerSilentAutofixNotification(List<Map<String,String>> sil
             }
         }
         catch(Exception exception){
-            LOGGER.error("exception occurred in triggerAutofixNotification with message - {}",exception.getMessage());
+            LOGGER.error("Error occurred in job triggerSilentAutofixNotification with message - {}",exception.getMessage());
```

Comment:
consider extracting this to a constant and standardizing the errors per same comments as for the collectors